src/test/regress/expected/multi_partitioning.out

```diff
@@ -417,6 +417,31 @@ SELECT * FROM partitioning_test WHERE id = 1 OR id = 2 ORDER BY 1;
   2 | 07-07-2010
 (2 rows)
 
+-- check the shard size
+SELECT * FROM citus_shard_cost_by_disk_size(1660000);
```

Comment:
try vacuum full on the table or drop index as index sizes might differ on PG 13.